src/emojis/emojiHandler.ts

```diff
@@ -0,0 +1,36 @@
+import emojis from "../helpers/files/emojis.json";
+
+export function getEmojisToReactWith(text: string): Array<string> {
+    const lowerCaseText = text.toLowerCase();
+
+    const keywords = Object.values(emojis).flat();
+
+    // search for each keyword in the text
+    const emojisToReactWith = findMatchingEmojiKeywords(
+        keywords,
+        lowerCaseText,
+        emojis
+    );
+
+    return Array.from(new Set(emojisToReactWith));
```

Comment:
the return type of `findMatchingEmojiKeywords` is `Array<string>`, do we still need to convert it to `Array`? 🤔🤔

Reply:
It is used to get only unique values from the array as Set removes duplicates and then again transform back to array.

Reply:
Ohh i see. Sounds good!